08-projects.Rmd

```diff
@@ -5100,9 +5100,42 @@ The following questions will use the dataset found in Scholar:
 
 A public sample of the data can be found [here](https://www.datadepot.rcac.purdue.edu/datamine/data/bay_area_bike_share/bay_area_bike_share.db).
```

Comment:
Hi @kevinamstutz , this link seems not to work. Could you double check and fix it (and for `p12-390`)? Thanks!